### PR TITLE
Programmatically set repo description and url

### PR DESCRIPTION
**Pull request by ab77**
Change-type: patch

### CODE REVIEW

**Comment by ab77** [comment]
@klutchell @alexgg I'v e never seen [this](https://github.com/balena-os/balena-raspberrypi/actions/runs/4256146113/jobs/7404661464#step:11:53) before, any ideas?

**Comment by klutchell** [comment]
> @klutchell @alexgg I'v e never seen [this](https://github.com/balena-os/balena-raspberrypi/actions/runs/4256146113/jobs/7404661464#step:11:53) before, any ideas?

@ab77 balenaOS device repos are "special" and cannot include a change-type, since the version is from meta-balena.

Last I checked the `Changelog-entry:` was a strict requirement on these repos too but I don't know if that's still the case.

**Comment by ab77** [comment]
> Last I checked the `Changelog-entry:` was a strict requirement on these repos too but I don't know if that's still the case.

Thank you. We also seem to have provision for reviewers in https://github.com/balena-os/balena-raspberrypi/blob/master/repo.yml#L3

We could get Flowzone to automatically configure this, if specified.

**Comment by klutchell** [comment]
Oh interesting, I wonder what VersionBot would have done with that info, or was it for resinCI?

File: .github/workflows/flowzone.yml [hunk 1]
```diff
@@ -19,3 +19,7 @@ jobs:
     name: Flowzone
     uses: product-os/flowzone/.github/workflows/flowzone.yml@master
     secrets: inherit
+    with:
+      repo_config: true
+      repo_description: "Balena support for RaspberryPI boards"
```
Comment (klutchell) [inline .github/workflows/flowzone.yml:24]:
```suggestion
      repo_description: "Balena support for Raspberry Pi boards"
```

**Pull request closed without merging**